src/cryptonote_core/cryptonote_format_utils.cpp

```diff
@@ -773,7 +807,7 @@ namespace cryptonote
   bool construct_tx(const account_keys& sender_account_keys, const std::vector<tx_source_entry>& sources, const std::vector<tx_destination_entry>& destinations, std::vector<uint8_t> extra, transaction& tx, uint64_t unlock_time)
   {
      crypto::secret_key tx_key;
-     return construct_tx_and_get_tx_key(sender_account_keys, sources, destinations, extra, tx, unlock_time, tx_key);
+     return construct_tx_and_get_tx_key(sender_account_keys, sources, destinations, extra, false, tx, unlock_time, tx_key);
```

Comment:
`construct_tx` seems to be used only by tests, and here I assume it's safe to pass `false` to the `is_onetime` parameter of `construct_tx_and_get_tx_key`.